src/clr-angular/data/datagrid/datagrid.ts

```diff
@@ -38,8 +39,18 @@ import {DatagridRenderOrganizer} from "./render/render-organizer";
     selector: "clr-datagrid",
     templateUrl: "./datagrid.html",
     providers: [
-        Selection, Sort, FiltersProvider, Page, Items, DatagridRenderOrganizer, RowActionService, ExpandableRowsCount,
-        HideableColumnService, StateDebouncer, StateProvider
+        Selection,
+        Sort,
+        FiltersProvider,
+        Page,
+        Items,
+        DatagridRenderOrganizer,
+        RowActionService,
+        ExpandableRowsCount,
+        HideableColumnService,
+        StateDebouncer,
+        StateProvider,
+        ColumnToggleButtonsService,
```

Comment:
trailing comma? do we care?

Reply:
I've come to love them myself. Later I can add a new item to this list without having a diff on the line above just to add a `,`. Our linting tools do not seem to mind either way though.